recipes-bsp/flashrom_git.bb

```diff
@@ -0,0 +1,19 @@
+DESCRIPTION = "flashrom is a utility for identifying, reading, writing, verifying and erasing flash chips"
+LICENSE = "GPLv2"
+HOMEPAGE = "https://github.com/pcengines/flashrom"
+
+LIC_FILES_CHKSUM = "file://COPYING;md5=751419260aa954499f7abaabaa882bbe"
+DEPENDS = "pciutils libusb libusb-compat"
+
+SRC_URI = "git://github.com/pcengines/flashrom.git \
+           "
+SRC_URI[md5sum] = "91bab6c072e38a493bb4eb673e4fe0d6"
+SRC_URI[sha256sum] = "aeada9c70c22421217c669356180c0deddd0b60876e63d2224e3260b90c14e19"
+
+S = "${WORKDIR}/flashrom-v${PV}"
+
```

Comment:
@Asiderr You can add custom PV to easily distinguish our custom flashrom. Something like PV = "1.0.1-pcengines+${SRCPV}" - depending on which flashrom version we are basing here.